pkg/util/connection.go

```diff
@@ -0,0 +1,136 @@
+package util
+
+import (
+	"fmt"
+	"io/ioutil"
+	"os"
+	"strings"
+
+	"github.com/openshift-online/uhc-cli/pkg/config"
+	"github.com/openshift-online/uhc-cli/pkg/dump"
+	"github.com/openshift-online/uhc-sdk-go/pkg/client"
```

Comment:
In other parts of the code the order of imports is the following:

```go
import (
        "standard-library-packages"

        "packages-from-other-projects"

        "packages-from-this-project"
)
```

So in this case it should be:

```go
import (
	"fmt"
	"io/ioutil"
	"os"
	"strings"

	"github.com/openshift-online/uhc-sdk-go/pkg/client"

	"github.com/openshift-online/uhc-cli/pkg/config"
	"github.com/openshift-online/uhc-cli/pkg/dump"
)
```

Can you do that change?